lib/engine/api/client.rb

```diff
@@ -43,9 +43,18 @@ def common_headers(api_key, config)
           'Authorization' => "Bearer #{api_key}",
           'SplitSDKVersion' => SplitIoClient::SplitFactory.sdk_version,
           'SplitSDKMachineName' => config.machine_name,
-          'SplitSDKMachineIP' => config.machine_ip
+          'SplitSDKMachineIP' => config.machine_ip,
+          'Referer' => referer
         }
       end
+
+      def referer
+        result = SplitIoClient::SplitFactory.sdk_version
+
+        result = "#{result}::#{SplitIoClient::SplitConfig.get_hostname}" unless SplitIoClient::SplitConfig.get_hostname == 'localhost'
```

Comment:
can this be nil?

Merging for now, but if the answer is yes, let's address it on development.